src/peer-connection-wrapper.cpp

```diff
@@ -520,7 +520,7 @@ Napi::Value PeerConnectionWrapper::createDataChannel(const Napi::CallbackInfo &i
                     init.reliability.rexmit = reliability.Get("rexmit").As<Napi::Number>();
                     break;
                 case 2:
-                    init.reliability.rexmit = std::chrono::milliseconds(reliability.Get("rexmit").As<Napi::Number>());
+                    init.reliability.rexmit = std::chrono::milliseconds((int)reliability.Get("rexmit").As<Napi::Number>().ToNumber());
```

Comment:
Just a nitpick: I think the C-style cast is weird here, a simple call to `Uint32Value()` should be sufficient.
```suggestion
                    init.reliability.rexmit = std::chrono::milliseconds(reliability.Get("rexmit").As<Napi::Number>().Uint32Value());
```

Reply:
I agree. using static_cast or Uint32Value call is better